sbncode/CAFMaker/FillReco.cxx

```diff
@@ -558,18 +558,19 @@ namespace caf
         p.dqdx = dqdx[i];
         p.dedx = dedx[i];
         p.pitch = pitch[i];
-        p.t = dprop.ConvertXToTicks(xyz[i].x(), calo.PlaneID());
-        p.p.x = xyz[i].x();
-        p.p.y = xyz[i].y();
-        p.p.z = xyz[i].z();
+        p.x = xyz[i].x();
+        p.y = xyz[i].y();
+        p.z = xyz[i].z();
 
         // lookup the wire -- the Calorimery object makes this
         // __way__ harder than it should be
         for (const art::Ptr<recob::Hit> &h: hits) {
           if (h.key() == tps[i]) {
             p.wire = h->WireID().Wire;
+            p.tpc = h->WireID().TPC;
             p.sumadc = h->SummedADC();
             p.integral = h->Integral();
+            p.t = h->PeakTime();
```

Comment:
@grayputnam do we understand why the previous version wasn't looking as intended? I think it seems ok to use the hit `PeakTime()` but I guess I'm not sure *why* the previous version was not right.

(I guess this for loop is looping through all the hits to find the one which matches the calo point presumably because there is no art::Assn between calo point and hit? -- Just double checking I understand the purpose of this loop.)

Reply:
Yes there is no association.

And I'm not sure why the x point method wasn't working... presumably Pandora is doing stuff with x position that I do not understand.

Reply:
Pandora can change the X position for example if it stitches the track across the cathode, but I don't see why it would impact things in the way that Maria was seeing.

Reply:
I don't either. But the time variable was always meant to encode the hit time. I was trying to be cute and get it out of Pandora. Regardless of whether that "should" work, directly accessing the hit time is correct.